openghg/retrieve/_access.py

```diff
@@ -19,7 +19,9 @@
 logger.setLevel(logging.DEBUG)  # Have to set level for logger as well as handler
 
 DataTypes = Union[BoundaryConditionsData, FluxData, FootprintData, ObsColumnData, ObsData]
```

Comment:
Quick question: just a thought but should we move these to the `openghg.types` sub-module and import them do you think? May be good to keep these in one place.

Reply:
I just know I've reinvented the wheel in the past before I discovered `openghg.types` but if you don't think they'll have broader applicability we can leave here.